sdk/ml/azure-ai-ml/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.18.0 (unreleased)
```

Comment:
Please also add an unreleased section for 1.19.0 at the top

Reply:
that I will adding after merging it to release branch